src/main/java/org/schemaspy/output/dot/schemaspy/name/Null.java

```diff
@@ -0,0 +1,16 @@
+package org.schemaspy.output.dot.schemaspy.name;
+
+/**
+ * Null object for name.
+ */
+public final class Null implements Name {
```

Comment:
Null is a loaded word, and it's actually an EmptyName. Could we rename it to EmptyName?

Reply:
Would `Empty` suffice ("name" can be inferred from the scope of the package and/or interface)?

Reply:
```
Name diagramName = new Degree(            
  twoDegreesOfSeparation,            
  new Implied(               
     includeImplied,                
     new Empty()           
  )        
);
```

Compilers, IDEs are better at inferring than humans. Empty is generic term so I think it hurts readability/understandability.

Import statement is at the top of the file.